### ISSUE TITLE
hello Louis 

### ISSUE THREAD

**Issue by ahlawy81** [LouisErigHerve/dsd#20]
I'm trying to work on EP / RC4 decryption and I wonder if it's get the iv from the signal frames or any header and if the keystream is only 40 bits (can't be less ) 
thanks in advance for your help

**Comment by LouisErigHerve** [comment]
Have a look on [DSD-FME](https://github.com/lwvmobile/dsd-fme/tree/pulseaudio), it decodes IV.

However after analyzing some parts of DSD-FME code, I'm a bit surprised because some functions are very, very similar to my private DSD code (same variables/functions name, same comment, same layout...) and I suspect this code has been stolen...

**Comment by lwvmobile** [comment]
> However after analyzing some parts of DSD-FME code, I'm a bit surprised because some functions are very, very similar to my private DSD code (same variables/functions name, same comment, same layout...) 

Its called copy and paste Louis. I recycled the functions already inside of dmr_sync.c to create new functions to handle other burst types, including the DMR PI Header, and that includes a lot of parts that aren't used but still have the same comments and so on. 

> and I suspect this code has been stolen...
I have no idea what you are referring to in regards to a private version, but if you suggest that anything is stolen, then may I please remind you of the license that the dmr_sync.c file has in its header. 

 * Permission to use, copy, modify, and/or distribute this software for any
 * purpose with or without fee is hereby granted, provided that the above
 * copyright notice and this permission notice appear in all copies.

**Comment by iScottybotty** [comment]
Boom!  😂

**Comment by iScottybotty** [comment]
> Have a look on [DSD-FME](https://github.com/lwvmobile/dsd-fme/tree/pulseaudio), it decodes IV.
> 
> 
> 
> However after analyzing some parts of DSD-FME code, I'm a bit surprised because some functions are very, very similar to my private DSD code (same variables/functions name, same comment, same layout...) and I suspect this code has been stolen...

Oh listen to her! 

Steady with the accusations and assumptions, Louis.

**Comment by iScottybotty** [comment]
Louis, bugger off! Your nothing but a wannabe who hides behind silly videos of what your version can, allegedly do. I personally have tried to make contact numerous times, YOUR NOT INTERESTED, you BLOCK everyone so, because of this we have all pushed to create something bigger and better than what you could ever achieve, and there's a lot more to come also! Your code is FULL of bugs and errors, and more so, very badly written. 

So, run along, create another pointless video of what you can do, then leave everyone hanging because you hide behind a "private" code. Idiot. 

Get your head out your arse, NO ONE GIVES A SHIT whether we can/ cannot decrypt BP, EP or any other kind of privacy and if you think Moto are gonna come running knocking my door, then you really do live a sad life with sad assumptions. Get real. Oh and Moto, if your reading this, bring it on! 

YOU created this, by being ignorant, by ignoring users, by making pointless clips and because your version, "isn't for public release". The public release version is useless. 

And it was ME that handed out YOUR CODE, and what? What are you actually going to do? Maybe I'll post a video about the LEH version and how much of a fraud you are, or should I say, "Monerioma". 

Bottom line is this Louis. Your package is POINTLESS if it does stuff you won't release. People don't appreciate smart arses, and you hiding away, blocking people and accusing people... enough said. 

Hide behind your version or release the full version with your BP and EP algorithms. If not, do one.

**Comment by ahlawy81** [comment]
It should not be public at all but you can provide it some educational purposes and only for our community even if it's paid, however it not illegal in all areas

**Comment by LouisErigHerve** [comment]
I close this issue now.
Any request concerning RC4/AES decryption shall be addressed ONLY over LinkedIn.
This type of paid service is reserved for government agencies, or companies specializing in radio-communications or cybersecurity registered in Europe.
A valid EU VAT number must be provided.
